src/PHPUnit/Constraint/WebDriverNot.php

```diff
@@ -53,5 +53,6 @@ public function toString(): string
         if ($this->string) {
             return 'that contains text "' . $this->string . '"';
         }
+        return '';
```

Comment:
I will try to move constraints from `src/PHPUnit/Constraint` to appropriate modules.